### ISSUE TITLE
Convert function takes string instead of type for conversion to

### ISSUE THREAD

**Issue by fubuloubu** [ethereum/vyper#1026]
### What's your issue about?
If you forget to add quotes to the type you want to convert to, Vyper will give an unhelpful error:
```python
convert(some_int128, uint256)
# raises:
#    AttributeError: 'Name' object has no attribute 's'
```

### How can it be fixed?
So, catching this error might work out to fix this issue, but I think the underlying issue is that the second argument (`convertTo`) is a string instead of a typename. This actually makes it a little more unintuitive to write conversions as you could misspell the name and not get visual feedback from your IDE (assuming you have syntax highlighting up)

I would suggest turning this into a VIP to modify the syntax of convert such that a valid typename is supplied as the second argument

#### Cute Animal Picture
![pony](https://upload.wikimedia.org/wikipedia/commons/thumb/8/8e/Gracie-rhs2005.jpg/1200px-Gracie-rhs2005.jpg)

**Comment by jacqueswww** [comment]
Yes agree, moving to convert(a, int128) without quotes is the way to go.

**Comment by jacqueswww** [comment]
Starting on this.

**Comment by fubuloubu** [comment]
cc @haydenadams
